cmd/storagenode/gracefulexit.go

```diff
@@ -139,3 +143,46 @@ func cmdGracefulExitInit(cmd *cobra.Command, args []string) error {
 	}
 	return nil
 }
+
+func cmdGracefulExitStatus(cmd *cobra.Command, args []string) error {
+	ctx, _ := process.Ctx(cmd)
+
+	ident, err := runCfg.Identity.Load()
+	if err != nil {
+		zap.S().Fatal(err)
+	} else {
+		zap.S().Info("Node ID: ", ident.ID)
+	}
+
+	client, err := dialGracefulExitClient(ctx, diagCfg.Server.PrivateAddress)
+	if err != nil {
+		return errs.Wrap(err)
+	}
+	defer func() {
+		if err := client.close(); err != nil {
+			zap.S().Debug("closing graceful exit client failed", err)
+		}
+	}()
+
+	// call get status to get status for all satellites' that are in exiting
+	progresses, err := client.getExitProgress(ctx)
+	if err != nil {
+		return err
+	}
+
+	if len(progresses) < 1 {
+		fmt.Println("No graceful exit in progress.")
```

Comment:
maybe this should be a `zap.S()`ed for consistency? not 100% sure though.

Reply:
this fails at compile time.  I think it should be `len(progresses.GetProgress())`